source/_posts/release-1.0.0-beta.3.md

```diff
@@ -13,6 +13,20 @@ Two smaller bugfixes releases have been tagged since `1.0.0-beta.0` so that some
 
 Even though we are still in our `beta` releases, we don't want to slow down our customers and keep on adding the features they need.
 
+### Improved embedded payments
+
+In previous versions, embedded payments such as Stripe or Payzen needed two steps to place an order : one to validate the payment's information, and another to place the order. We have improved this behavior and the user now only needs to submit its payment, and this will place the order directly.
```

Comment:
```suggestion
In previous versions, embedded payments such as Stripe or Payzen needed two clicks for placing an order: one to validate the payment's information, and another to place the order. We have improved this behavior and the user now only needs to submit its payment, and this will place the order directly in one click.
```